notification_delivery/config.py

```diff
@@ -29,6 +29,30 @@ class Config:
     DEBUG = False
     LIGHT_ID = '1'
 
+    # Mappings between Google Cloud alerting policy names
+    # and philips hue (color) values (0 to 65535). Each
+    # mapping indicates what hues the light bulb should
+    # light up when receiving a notification about an
+    # "open" or "closed" incident regarding a specific
+    # policy. The default mapping indicates the hues
+    # it should light up for notifications about "open"
+    # or "closed" incidents regarding any other unspecified
+    # policy.
+    POLICY_HUE_MAPPING = {
```

Comment:
Sidenote: Although using built-in types like Dicts and Sets are very common in Python, it can also be useful to define custom data types because it codifies the data structure (easy to read at a glance). The attrs library can cut down on boilerplate, which is probably a common reason why people default to the built-in types. See: https://www.attrs.org/en/stable/examples.html

I think that it's okay to submit as-is, but I just wanted to point out this alternative approach. Using built-in types is more typical in Python than say a language like C++ (probably because C++ has the added benefit of type safety). I'd say that dicts within dicts might be a borderline case, but probably still simple enough that many Python programmers would advocate against defining classes, which is why I think it's okay as is.

Reply:
Good to know, I'll definitely take this into account in the future.